packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -0,0 +1,117 @@
+@include base-component();
+
+:host {
+  @apply flex;
+  --calcite-meter-space-internal: theme("spacing[0.5]");
+  --calcite-meter-height-internal: theme("spacing.4");
+  --calcite-meter-font-size-internal: var(--calcite-font-size--1);
+}
+
+:host([scale="s"]) {
+  --calcite-meter-height-internal: theme("spacing.3");
+  --calcite-meter-font-size-internal: var(--calcite-font-size--2);
+}
+
+:host([scale="l"]) {
+  --calcite-meter-height-internal: theme("spacing.6");
+  --calcite-meter-font-size-internal: var(--calcite-font-size-0);
+}
+
+.meter {
+  @apply flex relative items-center w-full m-0;
+  block-size: var(--calcite-meter-height-internal);
+  background-color: var(--calcite-ui-foreground-2);
+  border: 1px solid var(--calcite-ui-border-3);
+  border-radius: var(--calcite-meter-height-internal);
+}
+
+.meter.solid {
```

Comment:
Would it be possible to use BEM-like classes along with the cascade to reduce specificity?